meetings/2024-09-27.md

```diff
@@ -0,0 +1,161 @@
+* PAC: Hesitant to remove since there are no many slots available. We will take it into account when we make a decision. 
+
+
+### Use cases input from participants
+
+* eP: What problems participants would like to solve with Solid?
+* eP: When there are different proposals to evaluate against the use cases. I'm not sure if there was anything submitted to the WG re use cases. How do we imagine organising those UCs, later when we evaluate the requirements, we can use those UCs. Some UCs are domain-specific. But requirements could be more generic. How would the WG imagine organising those UCs. As I imagine the UCs are left there. Whenever a proposal is evaluated it is checked against the UCs.
+* SC: Wednesday there was a [breaking silos meeting](https://docs.google.com/document/d/1sFZ0BozoLA3U7iROtu10tR7S05_OEOprS8Um9NZqZYw/edit), use cases from LWS, could PAC/others link to those meeting minutes? Are they published? That could be a response to the first question that eP had about which problems participants would like to solve. That's just a fraction of things. We have a long list of user stories issues and a repo but to my knowledge it is not properly comprehensive, there's some categories of use cases that are taken for granted: we assume they are covered by the requirements we have but we don't have anything more concrete, as eP suggested a proper matrix, something as to if use cases as evaluated, whether they're in scope, etc. 
+* HZ: One of hte big issues was in that call was about pod portability. It is very hard to move data from one location to another. If links are given, they may be broken. I'm not sure if that's something we should do in the WG or the CG.
+* eP: There was a meeting about portability is Social Web meeting. And there would be different proposal. One for custom domain names, anothr for different resolution mechanisms, e.g., that isn't based on dns. A matrix that requires 'those' and 'that' requirement. Also to see priorities. For me it is important, for every proposal, we can evaluate those UCs.
+* RG: What eP seems to ask is for architectural decision records in parallel with UCs and specifications. Perhaps something think in the WG.
+* SC: CG have some issues on this. There's an ocean of issues on portability as HZ pointed out. One on privisioning a data storage. Some of that had some question marks. There's a proposal made by ??? submitted to CG but we did not do a good job evaluating that, having to do with redirects and deleting. Some of that can be evaluated in the CG still. There's work that already had been done and if it seems mature we can handover to WG anytime. In the CG we have plenty of threads/considerations put into our radar and we need to revisit instead of assuming it's not there. 
```

Comment:
```suggestion
* SC: CG have some issues on this already. There's an ocean of issues on portability as HZ pointed out. One on provisioning a data storage ( https://github.com/solid/specification/issues/317 ). Some of that had some question marks. For example, there's a proposal made by PDS Interop folks for [Solid Migrator](https://github.com/pdsinterop/solid-migrator) submitted to CG - by MdJ in https://github.com/solid/specification/issues/474 - but we did not do a good job evaluating that, having to do with redirects and deleting. See also https://github.com/solid/specification/issues/136 , https://github.com/solid/specification/issues/462 . Some of that can be evaluated in the CG still. There's work that already had been done and if it seems mature we can handover to WG anytime. In the CG we have plenty of threads/considerations put into our radar and we need to revisit instead of assuming it's not there. 
```